students/ahwall/Lesson 02 Homework/FizzBuzz Exercise_HW2/FizzBuzzFinal.py

```diff
@@ -0,0 +1,18 @@
+'''
+This function uses a for loop with if elif else conditions to list numbers
+1 to 100.  If the number is divisible by 3, then Fizz is printed.  If divisible
+by 5, then Buzz is printed.  If divisible by both, FizzBuzz is printed.  The
+number is printed if none of the conditions are met.
+'''
+
```

Comment:
Wait, but you don't need to name files FizzBuzzFinal.py when you're using git! Just use the same file, save it, git add, and git commit. Then git push at the end of the day/hour! Use git to your advantage!